[2025-06-27T06:49:02.535+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:49:02.747+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-01-02T09:00:00+00:00 [queued]>
[2025-06-27T06:49:02.983+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-01-02T09:00:00+00:00 [queued]>
[2025-06-27T06:49:02.986+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:49:03.204+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-01-02 09:00:00+00:00
[2025-06-27T06:49:03.335+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=55) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:49:03.361+0000] {standard_task_runner.py:72} INFO - Started process 85 to run task
[2025-06-27T06:49:03.340+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_prod', 'start_task', 'scheduled__2025-01-02T09:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmp58uuf3s4']
[2025-06-27T06:49:03.426+0000] {standard_task_runner.py:105} INFO - Job 125: Subtask start_task
[2025-06-27T06:49:03.845+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-01-02T09:00:00+00:00 [running]> on host e7fdb6bf901c
[2025-06-27T06:49:04.107+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_prod' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-02T09:00:00+00:00'
[2025-06-27T06:49:04.122+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:49:04.132+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T06:49:04.133+0000] {logging_mixin.py:190} INFO - ds: 2025-01-02
[2025-06-27T06:49:04.136+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T06:49:04.162+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:49:04.163+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_prod, task_id=start_task, run_id=scheduled__2025-01-02T09:00:00+00:00, execution_date=20250102T090000, start_date=20250627T064902, end_date=20250627T064904
[2025-06-27T06:49:04.207+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:49:04.305+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T06:49:04.319+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
